giraffe/src/components/SimpleTable/PagedTable.tsx

```diff
@@ -109,26 +109,28 @@ const subsetResult = (
     }, {})
 
   const tables: SubsetTable[] = []
-  let lastTable
+  let lastTable = ''
 
   // group by table id (series)
   for (let ni = 0; ni < size; ni++) {
     if (
-      `y${subset['result'][0].data[ni]}:t${subset['table'][0].data[ni]}` ===
+      `y${subset['result']?.[0]?.data?.[ni]}:t${subset['table']?.[0]?.data?.[ni]}` ===
```

Comment:
It's not necessary in this PR, and I know you didn't initially write this code, but let's make sure we clean up some of these frankly atrocious variable names like `ni`, `y`, and `t`. Part of what makes this code so difficult to understand is that the variables are so poorly named, and have so much syntax (like `[`, `?`, `.`, `]`, etc).

It might also make sense to pull out common expressions into a named variable at the start of the loop, e.g.

```ts
for (let ni = 0; ni < size; ni++) {
const currentResult = subset['result']?.[0]?.data?.[ni]
const currentTable = subset['table']?.[0]?.data?.[ni]
```
(or some obviously better variable name if they're not actually the current result or current table. To my point about the variable names making this code really hard to follow, it's hard to even know what these things represent without diving deep into the code).